Add template to use and compare prior records from file (#62)
asreview/asreview-makita: README.md

# ASReview Makita

> [![PyPI version](https://badge.fury.io/py/asreview-makita.svg)](https://badge.fury.io/py/asreview-makita) [![Downloads](https://pepy.tech/badge/asreview-makita)](https://pepy.tech/project/asreview-makita) ![PyPI - License](https://img.shields.io/pypi/l/asreview-makita) [![DOI](https://zenodo.org/badge/530642619.svg)](https://zenodo.org/badge/latestdoi/530642619)[![test-suite](https://github.com/jteijema/asreview-makita/actions/workflows/ci-workflow.yml/badge.svg?branch=main)](https://github.com/jteijema/asreview-makita/actions/workflows/ci-workflow.yml)

Reviewer: ❓

asreview makita template multimodel --classifiers logistic nb --feature_extractors tfidf doc2vec --query_strategies max max_random max_uncertainty cluster --impossible_models nb,doc2vec logistic,tfidf
```

### Prior template

command: `prior`

> The prior template is used to evaluate how prior knowledge affects simulation performance. It processes all datasets from the `data` folder, grouping them into two: one containing datasets labeled `prior_[dataset_name]`, and another excluding these datasets. The template runs two simulations: the first includes both the prior and non-prior datasets, and the second uses only the non-prior datasets. This approach helps measure the impact of prior knowledge on simulation outcomes.

Reviewer 2: ```suggestion
The prior template evaluates how large amounts of prior knowledge might affect simulation performance. It processes two types of data in the data folder: one containing the dataset to be labeled and one (or more) dataset(s) containing the records used for prior knowledge. The filename(s) of the dataset(s) containing the priors should be titled  `prior_[dataset_name]`. The template runs two simulations: the first simulation uses all records from the prior dataset(s) as training data and the second uses a 1+1 randomly chosen set of prior knowledge. 
```

> Running this template also creates a `generated_data` folder. This folder will contain two datasets; `dataset_with_priors.csv` and `dataset_without_priors.csv`. The simulations specified in the generated jobs file will use these datasets for its simulations.

Reviewer 2: ```suggestion
Running this template creates a `generated_data` folder. This folder contains two datasets; `dataset_with_priors.csv` and `dataset_without_priors.csv`. The simulations specified in the generated jobs file will use these datasets for their simulations.
```

  --n_runs N_RUNS                           Number of runs.                                 Default: 1.
  --skip_wordclouds                         Disables the generation of wordclouds.
  --overwrite                               Automatically accepts all overwrite requests.
  --classifier CLASSIFIER                   Classifier to use.                              Default: nb.
  --feature_extractor FEATURE_EXTRACTOR     Feature_extractor to use.                       Default: tfidf.
  --query_strategy QUERY_STRATEGY           Query strategy to use.                          Default: max.
  --balance_strategy BALANCE_STRATEGY       Balance strategy to use.                        Default: double.
  --instances_per_query INSTANCES_PER_QUERY Number of instances per query.                  Default: 1.
  --stop_if STOP_IF                         The number of label actions to simulate.        Default 'min' will stop simulating when all relevant records are found.
```

#### Example usage

Put at least 2 datasets in the data folder. One starting with the `prior_` prefix, and one without this prefix. 

> > note: `priors_` will also work, testing showed that users try to use both.

Reviewer 2: ```suggestion
> note: `priors_` will also work.
```
